streisand-existing-cloud-server

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+#
+# Provision an existing cloud server.
+#
+# This requires an expanded extra-vars file specific to the provider type that
+# sets all of the values gathered by prompts in the interactive installation.
+# See the contents of global_vars/noninteractive for examples that can be copied
+# and modified.
+#
+# Usage:
+# $0 --provider digitalocean --site-config path/to/digitalocean-site.yml
```

Comment:
I think it would be clearer to replace `$0` with the current script filename since these comments are static and not something that can expand vars dynamically.